Cubical/Foundations/CartesianKanOps.agda

```diff
@@ -3,8 +3,7 @@
 module Cubical.Foundations.CartesianKanOps where
 
 open import Cubical.Foundations.Prelude
```

Comment:
Another observation: This module has no dependencies on the prelude, except for `refl`:

```suggestion
open import Cubical.Core.Everything

private
  refl : ∀ {ℓ} {A : Type ℓ} {x : A} → x ≡ x
  refl {x = x} _ = x
```

So in theory, it could go into `Cubical.Core.*` or be re-exported from `Cubical.Foundations.Prelude`. In any case, it is now possible to use the Kan operations inside of `Cubical.Foundations.*` without introducing dependency cycles. This would perhaps clean up a few ad-hoc uses of `coe0→i` and friends, such as in [isEquivTransp](https://agda.github.io/cubical/Cubical.Foundations.Equiv.Base.html#isEquivTransp).